src/wave_tank/Makefile.am

```diff
@@ -0,0 +1,3 @@
+## Process this file with automake to produce Makefile.in
+include $(top_srcdir)/config/Make-rules
+SUBDIRS = .
```

Comment:
This doesn't make any sense: the current directory is not a subdirectory.

Reply:
Are you suggesting to remove SUBDIRS all together or just the '.' or the current directory from Makefile.am?

Reply:
You should be able to remove it completely.